editor/src/clj/editor/cljfx_form_view.clj

```diff
@@ -277,12 +282,12 @@
 
 ;; region integer input
 
-(defmethod form-input-view :integer [{:keys [value on-value-changed] :as field}]
+(defmethod form-input-view :integer [{:keys [value on-value-changed custom-max-width] :as field}]
```

Comment:
in case the field is in table cell, we want to use more cell size if it's available that a regular minimum size.